canopy/design-system/README.md

```diff
@@ -0,0 +1,89 @@
+This project was bootstrapped with
```

Comment:
Is this README.md content actually useful? Looks like maybe it was autogenerated and mostly points to facebook URLs and isn't really canopy/design-system specific.

Reply:
It is the generic create-react-app README and will get updated later but it does, for now, list the relevant commands to run the app